bookwyrm/templates/import/import.html

```diff
@@ -15,6 +15,12 @@ <h1 class="title">{% trans "Import Books" %}</h1>
     {% endif %}
 
     {% if site.imports_enabled %}
+        {% if import_size_limit and import_limit_reset %}
+            <div class="notification">
+                <p>Currently you are allowed to import {{ import_size_limit }} books every {{ import_limit_reset }} days.</p>
```

Comment:
These blocks of text will need to be wrapped in `{% blocktrans %]` tags so they can be localized. For example:
```
<p>{% blocktrans %}Currently you are allowed to import {{ import_size_limit }} books every {{ import_limit_reset }} days.{% endblocktrans %}</p>
```